### ISSUE TITLE
Add option to the proxy to return HTTP/1.1 responses instead of HTTP/1.0

### ISSUE THREAD

**Issue by rodolfo2488** [cloudfoundry-incubator/haproxy-boshrelease#136]
Hi Team!

I am dealing with an issue that HAProxy is returning Error 503 Unavailable responses as HTTP/1.0

I wonder if we could add a section for `errorfile` in the config to send custom error responses so we can specify the version of the response. [Docs](https://cbonte.github.io/haproxy-dconv/1.7/configuration.html#4-errorfile)

The main reason to upgrade to HTTP/1.1 is the way `keep-alive` connections are handled.

Example of the errorfile:
```
HTTP/1.1 503 Service Unavailable
Cache-Control: no-cache
Connection: close
Content-Type: text/html

<html><body><h1>503 Service Unavailable</h1>
No server is available to handle this request.
</body></html>
```

**Comment by geofffranks** [comment]
Sure, feel free to submit a PR :)

**Comment by rodolfo2488** [comment]
I will address design discussions here: https://github.com/cloudfoundry-incubator/haproxy-boshrelease/pull/137

I thought I signed the ACL but it seems like I did not. I will do that too! sorry about it

**Comment by geofffranks** [comment]
Fixed vai #137
